Formula/ib.rb

```diff
@@ -0,0 +1,17 @@
+class Ib < Formula
+  desc "C/C++ build tool that automatically resolves included source"
+  homepage "https://github.com/JasonL9000/ib"
+  url "https://github.com/hooddanielc/ib/archive/v0.7.tar.gz"
```

Comment:
We need an official release published here: https://github.com/JasonL9000/ib/releases